### ISSUE TITLE
Linux support

### ISSUE THREAD

**Issue by piotrekkaminski** [magento/magento2devbox-web#18]
Please +1 in this thread if you would like to see Linux support.

**Comment by kesonno** [comment]
+1

**Comment by mbitson** [comment]
+1

**Comment by DerekMarcinyshyn** [comment]
+1

**Comment by piotrekkaminski** [comment]
@kesonno @mbitson @DerekMarcinyshyn any specific flavor - helps us with testing (Ubuntu,CentOS, RedHat etc)?

**Comment by mbitson** [comment]
@piotrekkaminski Ubuntu (14.04 & 16.04) for us.

**Comment by sean-jamersan** [comment]
+1

**Comment by JasonTipton-Jamersan** [comment]
+1

**Comment by sean-jamersan** [comment]
Will help test Ubuntu 16.04

**Comment by schmengler** [comment]
Arch Linux

**Comment by avstudnitz** [comment]
+1 Ubuntu 16.04

**Comment by paras89** [comment]
+1 Ubuntu 14.04

**Comment by roman204** [comment]
+1 Ubuntu 16.04

**Comment by barbanet** [comment]
+1 Ubuntu 14.04 & 16.04

**Comment by talesh** [comment]
+1 Fedora

**Comment by slackerzz** [comment]
+1

Ubuntu 16.04

**Comment by fooman** [comment]
FYI if I have just been able to download the Mac version and have a Magento 2 instance running on Linux (OpenSuse).

I had some permission issues with the folders it wanted to share from the home directoy - so I have commented out in docker-compose.yml the volumes `~/.composer` and `~/.ssh`

**Comment by piotrekkaminski** [comment]
yes, this was our experience as well - permission issues. Because very few people in the office use Linux, we just did not have time to test it properly @fooman

**Comment by fooman** [comment]
Docker for Mac automatically makes sharing volumes easier. As this is different on Linux I believe the issue could be that it runs under user magento2 which would likely conflict with the actual owner of the files. So maybe just making the .composer folder a new one under shared would help. And I don't believe it needs access to .ssh.

**Comment by mrdevries** [comment]
+1 (Ubuntu 16.04)

**Comment by ArneFfm** [comment]
+1 Ubuntu 16.04

**Comment by denispapec** [comment]
+1 Arch Linux

**Comment by jbasoo** [comment]
+1 Ubuntu 14.04

**Comment by mrtuvn** [comment]
+1 ubuntu

**Comment by beejhuff** [comment]
+1 Ubuntu

**Comment by davidmpaz** [comment]
+1 Ubuntu 14.04 

Don't think this should be asked if the intention was to create a some how standard development environment based in Docker, which has had native support for Linux since the beginning ;)

**Comment by ebaschiera** [comment]
+1 Ubuntu 16.04

**Comment by caresposito79** [comment]
+1 on Ubuntu 16.04 with fixed version of Magento 2

**Comment by Kianda** [comment]
+1 Ubuntu 16.04

**Comment by glongshadow** [comment]
+1 Kubuntu 16.04

**Comment by slirx** [comment]
+1 Arch Linux

**Comment by johnhughes1984** [comment]
+1 Ubuntu 16.04

**Comment by ClaudiuCreanga** [comment]
+1 Ubuntu 16.04

**Comment by julienloizelet** [comment]
+1 Ubuntu 14.04

**Comment by holozaen** [comment]
+1 ubuntu 14.04

**Comment by chrom** [comment]
+1
OpenSUSE

**Comment by piotrekkaminski** [comment]
Wow, 32 +1s so far. Ok - definitely adding to backlog.

**Comment by amarroni** [comment]
+1 Ubuntu 14.04 & 16.04

**Comment by dzschille** [comment]
+1 Ubuntu 16.04

**Comment by jzahedieh** [comment]
+1 Ubuntu 16.04

**Comment by waynetheisinger** [comment]
+1 Ubuntu 16.04

**Comment by sergi-omit** [comment]
+1 Ubuntu 14.04

**Comment by mrtuvn** [comment]
+1 Ubuntu

**Comment by joesken** [comment]
Ubuntu 16.04 as well. As per above ran into a tangle of permissions issues trying to get it running at the moment.

**Comment by MatthieuMota** [comment]
+1 Arch Linux & Ubuntu 16.04

**Comment by jandro-deporvillage** [comment]
+1 Ubuntu 16.04

**Comment by davidvarney** [comment]
+1 Ubuntu 14.04/16.04

**Comment by dannydnz** [comment]
+1 Ubuntu 14.04/16.04

**Comment by equit** [comment]
+1 Ubuntu

**Comment by Pakkapao** [comment]
+1 Ubuntu 16.04

**Comment by sergi-omit** [comment]
Any new @piotrekkaminski?

**Comment by piotrekkaminski** [comment]
@sergi-omit unfortunately not yet

**Comment by michaeldyl520** [comment]
Can I get it for linux now?

**Comment by OSdave** [comment]
+1 Ubuntu 16.04

**Comment by 0m3r** [comment]
+1 Ubuntu 16.04

**Comment by mrda** [comment]
+1 Centos 7

**Comment by yuryostapenko** [comment]
+1 Centos 7

**Comment by phirunson** [comment]
+1 Centos 7.3

**Comment by hazrpg** [comment]
+1 Ubuntu 16.04

**Comment by henkelund** [comment]
+1 Debian Stretch

**Comment by jonpday** [comment]
ping @piotrekkaminski @benmarks @dougkm @gregwork any updates please?

**Comment by mealiusman** [comment]
+1 Arch Linux & Ubuntu 16.04

Any news?

**Comment by kaushik-chavda** [comment]
+1 Ubuntu 16.04

**Comment by hnhtag** [comment]
Ubuntu 16.04.4 LTS

**Comment by ctsexton** [comment]
+1 Arch linux, any word on ETA?

**Comment by piotrekkaminski** [comment]
As easy to notice, this project is dead. We had too many issues with file syncing. I'd like to provide you with two actively maintained alternatives that we have success in using:
- https://github.com/paliarush/magento2-vagrant-for-developers 
- https://inchoo.net/dev-talk/local-development-with-valet/

**Comment by Outdoorsman** [comment]
Well just thought I'd mention that there is also DDEV. I've texted with the devs and sounds like it works for pretty much any PHP project and they may be adding specific support for Magento here in the future too. Suuuuper easy to use. We're using it for a TYPO3 project.

https://www.drud.com/
https://github.com/drud/ddev

**Comment by salehawal** [comment]
it is strange that linux is not supported?
